modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -807,7 +812,7 @@ public void setSwiftUseApiNamespace(boolean swiftUseApiNamespace) {
     @Override
     public String toEnumValue(String value, String datatype) {
         // for string, array of string
-        if ("String".equals(datatype) || "[String]".equals(datatype) || "[String: String]".equals(datatype)) {
```

Comment:
Could you explain this change please?

Reply:
[Character](https://developer.apple.com/documentation/swift/character) uses quotes when initialized same as strings it needs to be surrounded by quotes `"` when defined as an enum.

Reply:
So this is a bug fix? It's not working in the current release?

Reply:
Commit dropped. I was trying something out and forgot to remove this line.